org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/codeactions/LSPCodeActionsMenu.java

```diff
@@ -103,13 +103,15 @@ public void fill(final Menu menu, int index) {
 		super.fill(menu, index);
 	}
 
-	private void scheduleMenuUpdate(final Menu menu, final MenuItem placeHolder, final int index, final IDocument document, final LanguageServerWrapper wrapper, final Throwable u, final List<Either<Command, CodeAction>> codeActions) {
+	private void scheduleMenuUpdate(final Menu menu, final MenuItem placeHolder, final int index,
+			final IDocument document, final LanguageServerWrapper wrapper, final @Nullable Throwable ex,
+			final @Nullable List<@Nullable Either<Command, CodeAction>> codeActions) {
 		final var job = new UIJob(menu.getDisplay(), Messages.updateCodeActions_menu) {
 			@Override
-			public IStatus runInUIThread(IProgressMonitor monitor) {
-				if (u != null) {
+			public IStatus runInUIThread(@Nullable IProgressMonitor monitor) {
+				if (ex != null) {
 					final var item = new MenuItem(menu, SWT.NONE, index);
-					item.setText(u.getMessage());
+					item.setText(String.valueOf(ex.getMessage()));
```

Comment:
why do we add `String.valueOf`? Could we not leave it as it was?

Reply:
get Message is nullable but setText expects a non null. value